src/main.c

```diff
@@ -127,16 +127,15 @@ main (int argc, char *argv[])
 
   }
   while (next_option != -1);
-
-  if (getenv ("HOME") == NULL)
+  
+  char *HOMEDIR = getenv ("HOME");
+  
+  if (HOMEDIR == NULL)
```

Comment:
not yet but i will install it soon .. 
```c
if (HOMEDIR == NULL)
  {
    fprintf (stderr, "Error: Environmental variable $HOME can't be used. Unable to determine home directory\n");
    return 1;
  }
```
And we are printing that error when HOMEDIR ==NULL but in above code block it is opposite